### ISSUE TITLE
Docs: how to use OpenAPI connector

### ISSUE THREAD

**Issue by bajtos** [strongloop/loopback-next#4488]
As a LoopBack 4 user connecting to a REST service with an OpenAPI description, I would like to leverage [loopback-connector-openapi](https://github.com/strongloop/loopback-connector-openapi) to build an easy-to-use service proxy for me. Unfortunately, there is very little information how to do that.

## Acceptance Criteria

- [x] Enhance [Calling other APIs and web services](https://loopback.io/doc/en/lb4/Calling-other-APIs-and-web-services.html) to show how to use OpenAPI connector
- [x] Update connector's [README](https://github.com/strongloop/loopback-connector-openapi/blob/master/README.md) file to make it relevant for LB4 users.

Possible structure:
- In "Calling other APIs", provide short section covering the common path at high level and point users to the connector docs for more details.
- In connector's README, go deeper into OpenAPI-specific configuration and behavior.

**Comment by dhmlau** [comment]
@bajtos @raymondfeng , just wanna confirm with you the requirement of this task before starting to write anything. 

If we want to call a third party REST API, say the [PetStore](http://petstore.swagger.io:8080/), we would:
- step 1: Run `lb4 openapi` with the OpenAPI json, to create the models and controller
- step 2: Run `lb4 datasource` with the PetStore API url using REST connector
- step 3: Run `lb4 service` to create the service proxy with the above datasource
- step 4: Implement the methods in the controller generated in step 1 to call the service from step 3

Are they what the right steps? Thanks.

**Comment by raymondfeng** [comment]
`lb4 openapi` is to implement an OpenAPI spec. To access 3rd party REST service with OpenAPI spec, use `lb4 datasource` and `lb4 service` (with [loopback-connector-openapi](https://github.com/strongloop/loopback-connector-openapi)).

In the future, we should extend `lb4 openapi --client` option to generate client service proxies with strongly-typed interfaces and types. Contrast to `lb4 openapi` generating controller code, the client option should generate `client proxy/stub` instead.

**Comment by dhmlau** [comment]
I actually tried with `lb4 datasource` (with OpenAPI connector) and `lb4 service`. 
The datasource json is:
```{
  "name": "ds",
  "connector": "openapi",
  "spec": "http://petstore.swagger.io:8080/api/v3/openapi.json",
  "validate": false,
  "positional": false
}

```

and I ran into error:
```
Unhandled error in POST /pet: 500 Error: only absolute urls are supported
```

Any ideas?

**Comment by dhmlau** [comment]
Let me try again and create a repo if the problem still exists.

**Comment by dhmlau** [comment]
@raymondfeng, looks like the "absolute urls" error still exists. 

This is my repo: https://github.com/dhmlau/openapi-test2
- there is one endpoint for addPet. Use the following requestBody:
```


{
  "id": 10,
  "name": "doggie",
  "category": {
    "id": 1,
    "name": "Dogs"
  },
  "photoUrls": [
    "string"
  ],
  "tags": [
    {
      "id": 0,
      "name": "string"
    }
  ],
  "status": "available"
}
```

Then you'll get an error something like:
```
Unhandled error in POST /pets: 500 Error: only absolute urls are supported
    at /Users/dhmlau/loopback4-example/openapi-test/node_modules/node-fetch/index.js:54:10
    at new Promise (<anonymous>)
    at new Fetch (/Users/dhmlau/loopback4-example/openapi-test/node_modules/node-fetch/index.js:49:9)
    at Object.Fetch (/Users/dhmlau/loopback4-example/openapi-test/node_modules/node-fetch/index.js:37:10)
    at Object.fetch (/Users/dhmlau/loopback4-example/openapi-test/node_modules/cross-fetch/fetch-node.js:11:20)
    at _callee$ (/Users/dhmlau/loopback4-example/openapi-test/node_modules/swagger-client/dist/index.js:581:86)
```
Thanks!

**Comment by dhmlau** [comment]
Still investigating. Got passed the above error, because it's expecting url in https://github.com/dhmlau/openapi-test2/blob/master/petstore.json#L18 to be in absolute path. But I got a 400 Bad Request error.

**Comment by dhmlau** [comment]
Got it working in https://github.com/dhmlau/openapi-test2. 
Taking the petstore json, I need to modify this line:
```
  "servers": [{"url": "/api/v3"}],
```

to 
```
  "servers": [{"url": "http://petstore.swagger.io:8080/api/v3/"}],
```
Otherwise, I will get the `500 Error: only absolute urls are supported` error.

**Comment by dhmlau** [comment]
Closing as done.

**Comment by kannant-sf** [comment]
@dhmlau  am facing issue while creating post, i don't find any proper documentation on how to call post method and how many params we need to send in which param we need to send body data?

**Comment by kannan007** [comment]
@dhmlau  i saw the way you had written in controllers and service file, tried the same in my code but am getting 400 status code error
